php/php.editor/src/org/netbeans/modules/php/editor/actions/ImportData.java

```diff
@@ -63,13 +64,14 @@ public static class DataItem {
         private final String typeName;
         private final List<ItemVariant> variants;
         private final ItemVariant defaultVariant;
+        @NonNull
```

Comment:
Nitpick: in NetBeans, all variables/parameters are not null, by default. Therefore, usually only `@Nullable` ones are annotated.

Reply:
I see. I'll remove it. Thanks!

Reply:
Fixed.